src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/ProviderRelationship/ProviderRelationshipsService.cs

```diff
@@ -32,7 +32,39 @@ public class ProviderRelationshipsService : IProviderRelationshipsService
 
         public async Task<IEnumerable<EmployerInfo>> GetLegalEntitiesForProviderAsync(long ukprn)
         {
-            ProviderPermissions providerPermissions = null;
+            var providerPermissions = await GetProviderPermissionsAsync(ukprn);
```

Comment:
(see my comment on `GetProviderPermissionsAsync` first) I think if `GetProviderPermissionsAsync` is updated to return empty list, then the check for `.Any()` will work here.

Reply:
👍 I can remove the below line below entirely